packages/hoppscotch-common/src/components/accessTokens/List.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+  <div class="grid gap-4 sm:grid-cols-2 md:grid-cols-3 lg:grid-cols-4">
+    <div
+      v-for="{ id, label, lastUsedOn, expiresOn } in accessTokens"
+      :key="id"
+      class="flex items-center justify-between p-4 border rounded md:items-baseline md:flex-col md:gap-4 md:justify-normal border-divider"
+    >
+      <div class="w-full text-sm font-semibold truncate text-secondaryDark">
+        {{ label }}
+      </div>
+
+      <div class="flex items-center gap-x-4">
```

Comment:
```suggestion
    <div class="grid p-4 gap-4 sm:grid-cols-2 md:grid-cols-3 xl:grid-cols-4">
    <div
      v-for="{ id, label, lastUsedOn, expiresOn } in accessTokens"
      :key="id"
      class="flex items-center justify-between p-4 border rounded flex-col gap-4 border-divider"
    >
      <div class="w-full text-sm font-semibold truncate text-secondaryDark">
        {{ label }}
      </div>

      <div class="flex items-center w-full justify-between gap-x-4">
```

we can justify the Delete button , flex-col for smaller screens and set 3 cards for larger to xl screens..

Reply:
Good catch. Also, a 3-column layout didn't fit well for tablet devices falling in a breakpoint b/w `md` & `lg`. Hence, it is now applied only from the `lg` breakpoint.

> Before

  ![image](https://github.com/hoppscotch/hoppscotch/assets/25279263/7e313412-8d70-4f84-8d59-b999ac8d329f)

> After

  ![image](https://github.com/hoppscotch/hoppscotch/assets/25279263/81a66957-50d8-419e-a0c0-1b43f1abd9bc)